observation_portal/requestgroups/serializers.py

```diff
@@ -153,13 +152,15 @@ def validate(self, config_dict) -> dict:
             ))
         return validated_config_dict
 
-    def mode_exists(self, mode_value: str) -> bool:
+    def mode_exists_and_is_schedulable(self, mode_value: str) -> bool:
```

Comment:
Check if both this mode exists, and if it does that it is schedulable. The error is the same either way if not, that the mode is not available.